app/utilities.py

```diff
@@ -3,64 +3,33 @@
 from flask import jsonify
 
 
-def validate_login(args):
-    """Function to validate user provided fields for login"""
-    params = ['email', 'password']
-    length = 2
-
-    # Check provided fields
-    if check_keys(args, params, length):
-        return check_keys(args, params, length)
-
-
-def validate_signup(args):
-    """Function to validate user-provided info when signing up"""
-    params = ['username', 'email', 'password', 'confirm-password']
-    length = 4
-
-    # Check fields provided
-    if check_keys(args, params, length):
-        return check_keys(args, params, length)
-
-
 def validate_email(args):
     """Function to validate username"""
-    if re.match(r'^[a-zA-Z0-9_\-\.]{3,}@.*\.[a-z]{2,4}$', args["email"]):
-        return True
-    return False
+    if not re.match(r'^[a-zA-Z0-9_\-\.]{3,}@.*\.[a-z]{2,4}$', args["email"]):
+        msg = "please enter valid email"
+        response = jsonify({"error": msg})
+        response.status_code = 400
+        return response
 
 
 def validate_username(args):
     """Function to validate username"""
-    if re.match(r'^[a-zA-Z0-9]{5,20}$', args["username"]):
-        return True
-    return False
+    if not re.match(r'^[a-zA-Z0-9]{5,20}$', args["username"]):
+        msg1 = "username should have "
```

Comment:
can do 
```python
msg = "username should have "
msg += "username should have "
msg += "username should have "

----or----
msg = "username should have \
username should have \
username should have "

---or---
msg = '''
username should have 
username should have
username should have 
'''

rather than creating three separate variables

Reply:
Thanks @Dhul. I have gone with the latter option.